packages/web-frontend/cypress/constants.js

```diff
@@ -5,7 +5,6 @@
 
 import { integrationFolder } from '../cypress.json';
 
-export const CONFIG_ROOT = 'cypress/config';
```

Comment:
Initially I thought that it would would be useful to encapsulate where config is stored. In practice, we use a lot of hard coded `config/...` imports, so no reason to pretend that this is configurable.